framework/include/cppmicroservices/BundleInitialization.h

```diff
@@ -60,12 +60,16 @@ class BundleContextPrivate;
     std::atomic<cppmicroservices::BundleContextPrivate*> US_CTX_INS(           \
       US_BUNDLE_NAME){};                                                       \
                                                                                \
+    extern "C" cppmicroservices::BundleContextPrivate* US_GET_CTX_FUNC(        \
```

Comment:
Add single line comment to explain why the declaration is included

Reply:
Deciding not to do this, the inclusion of that line should be obvious (regarding what it syntactically means (prototype definition)) (even if somebody doesn't know what warning we are trying to avoid)